package com.jdluu.leafline.library.data.local

import androidx.room.Database
import androidx.room.RoomDatabase
import androidx.room.TypeConverters

@Database(
    entities = [BookEntity::class],
    version = 1,
    exportSchema = true
)
@TypeConverters(Converters::class)
abstract class LeaflineDatabase : RoomDatabase() {
    abstract fun bookDao(): BookDao
}